multierror/multierror.go

```diff
@@ -0,0 +1,28 @@
+package multierror
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Error []error
+
+func (me Error) Error() string {
+	if me == nil {
+		return ""
+	}
+
+	strs := make([]string, len(me))
+	for i, err := range me {
+		strs[i] = fmt.Sprintf("[%d] %v", i, err)
+	}
+	return strings.Join(strs, " ")
+}
+
+func (me Error) ErrorOrNil() error {
```

Comment:
Not that I can think of. This is exactly what I would expect, as I I haven't used either of these packages before and I went looking for a method that does this, and it was here.